sos/report/plugins/kata_containers.py

```diff
@@ -43,9 +43,7 @@ def setup(self):
 
             self.add_copy_spec(config_files)
 
-        self.add_journal(identifier='kata-proxy')
-        self.add_journal(identifier='kata-shim')
-        self.add_journal(identifier='kata-runtime')
-        self.add_journal(units='kata-ksm-throttler')
+        self.add_journal(identifier='crio')
```

Comment:
The `crio` plugin already captures the `crio` journal.

Reply:
So, I'd also remove the `containerd` journal and leave that for a possible `containerd` plugin.  This simplifies even more things on the kata-containers side. :-)